acvm-repo/acvm_js/test/browser/execute_circuit.test.ts

```diff
@@ -7,7 +7,7 @@ import initACVM, {
   WitnessMap,
   initLogLevel,
   ForeignCallHandler,
-} from "../../../result/";
+} from "@noir-lang/acvm_js";
```

Comment:
This still pulls in the local package version?

Reply:
We could also do `../../web/index.js` but this seems cleaner and ensures that `package.json` resolves it correctly.